Source/Example/Catalog/Web/Localization/ExampleStringManager.cs

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) 2017 International Federation of Red Cross. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+ 
+ namespace Web.Localization
+{
+    public class ExampleStringManager
```

Comment:
Suggest not suffixing this with Manager - as it is not managing anything. So just "ExampleStrings".